vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -304,6 +305,81 @@
             		$('#file').prop("disabled",true);
             		$('#imageId').prop("disabled",false);
             	  });
+            	var timeout=null;
+            	$('#slideSpace').sortable({
+	            	update: function(event,ui) {
+		            	var jsonObj = [];
+		            	var blockId;
+		            	$(this).children().each(function(index,element){
+			            	if(typeof($(element.innerHTML).filter('div').attr('data-position'))=="undefined") {
+			            		blockId=$(element).attr('id');
+			            	}else {
+			            		blockId = $(element.innerHTML).filter('div').attr('id');
+			            	}
+			            	item = {}
+			            	item ["id"] = blockId;
+			            	item ["contentOrder"] = index;
+			            	jsonObj.push(item);
+			            	$(element.innerHTML).filter('div').attr('data-position',index);
+		            	});
+		            	var token = [[${_csrf.token}]];
+		            	if (jsonObj.length != 0){
+			            	if (timeout == null){
+			            		timeout = setTimeout(updateContentOrder(jsonObj,token), 1000);
+			            	}
+			            	else{
+			            		clearTimeout(timeout);
+			            		timeout = setTimeout(updateContentOrder(jsonObj,token), 1000);
```

Comment:
this line is the same as in the if block so it should be outside of the if/else statement (DRY).